fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -335,19 +335,19 @@ def apply_trunk_native_vlan(self, interface_data, port):
     def get_trunk_native_vlan_node(self, interface_node):
         return interface_node.xpath("unit/family/ethernet-switching/native-vlan-id")
 
+    @staticmethod
```

Comment:
i mean why static?
I get the "it doesn't use self" argument, but philsophilcally, it does not act on the class, of the instances but rather a a simple extraction of an instance method, i don't think that qualifies it to be static.  and maybe it's also worth being _underscored as a protected method